specification/security/resource-manager/Microsoft.Security/stable/2019-01-01/examples/RegulatoryCompliance/getRegulatoryComplianceAssessmentList_example.json

```diff
@@ -0,0 +1,59 @@
+{
+    "parameters": {
+		"api-version": "2019-01-01",
+		"scope": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+		"regulatoryComplianceStandardName": "PCI-DSS-3.2",
+		"regulatoryComplianceControlName": "1.1"
+    },
+    "responses": {
+		"200": {
+			"body": {
+				"value": [{
+					"id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/regulatoryComplianceStandards/PCI-DSS-3.2/regulatoryComplianceControls/1.1/regulatoryComplianceAssessments",

```

Comment:
id value is incorrect. it does not include the name of the regulatorycomplianceAsssement